azure/terraform/dbadministrators.rego

```diff
@@ -20,39 +20,37 @@ azure_issue["db_ad_admin"] {
     lower(resource.type) == "azurerm_sql_server"
     count([c | r := input.resources[_];
                r.type == "azurerm_sql_active_directory_administrator";
-               re_match(concat("", ["^.*\\.", resource.name, "\\..*$"]), r.properties.server_name);
+               re_match(concat("", ["^.*\\.", resource.name, "\\..*$"]), r.properties.server_name); # Rezoan: this regex is not correct and need a fix. Snapshot file does not generate server_name out of the tf variable as well at r.properties.server_name
```

Comment:
@rezoan assign a task to @vatsalgit5118 to fix the generated snapshot

Reply:
@farchide bugs has already been reported 
- https://github.com/prancer-io/prancer-compliance-test/issues/138
- https://github.com/prancer-io/prancer-compliance-test/issues/139